languages/csharp/concept-exercises/pattern-matching/.docs/hints.md

```diff
@@ -0,0 +1,28 @@
+﻿# Hints
+
+### 1. Check the type 
```

Comment:
I'll skip reviewing this file due to the exercise possibly changing due to a comment I made elsewhere.